Description: do not build tools with rpath
 These will be installed semi-manual to usr/bin and will use system libraries.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: not-needed
Last-Update: 2021-10-16

---

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1377,6 +1377,7 @@ if(WITH_TESTS)
 endif()
 
 if(WITH_BENCHMARK_TOOLS)
+  set(CMAKE_SKIP_BUILD_RPATH TRUE)
   add_executable(db_bench${ARTIFACT_SUFFIX}
     tools/simulated_hybrid_file_system.cc
     tools/db_bench.cc
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -1,3 +1,5 @@
+set(CMAKE_SKIP_BUILD_RPATH TRUE)
+
 set(CORE_TOOLS
   sst_dump.cc
   ldb.cc)
